lncfg/protocol.go

```diff
@@ -18,8 +20,9 @@ type ProtocolOptions struct {
 	// mini.
 	WumboChans bool `long:"wumbo-channels" description:"if set, then lnd will create and accept requests for channels larger chan 0.16 BTC"`
 
-	// Anchors enables anchor commitments.
-	Anchors bool `long:"anchors" description:"enable support for anchor commitments"`
+	// NoAnchors should be set if we don't want to support opening or accepting
+	// channels having the anchor commitment type.
+	NoAnchors bool `long:"no-anchors" description:"disable support for anchor commitments"`
```

Comment:
Just a thought, that it would be nice to not change this field but somehow leave it default to true.

Reply:
Unfortunately, IIRC, the flag parser we use cannot handle command line flags being `false`. So if we set this this true by default it would be no way of opting-out.